.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+name: Python application
+
+on:
+  push:
+    branches:
+      - main
+      - develop
```

Comment:
@arzukhanna 

For the moment - add your feature branch name here as well.

- [ ] Why?
- [ ] What do we have to do to make this build on every feature branch?

Hint: https://docs.github.com/en/actions/reference/events-that-trigger-workflows

## Challenge

- [ ] condition the job so that `check` and `test` always execute, but `test` only runs on `develop` & `main`

Hint: https://kevsoft.net/2020/06/10/running-github-action-steps-and-jobs-only-on-push-to-master.html

Reply:
Why? - So that the pipeline runs every time a change is pushed to the feature branch
To build on every branch do we add 'feature/*' ?